src/test/javascript/spec/component/exam/manage/exam-management.component.spec.ts

```diff
@@ -84,4 +85,53 @@ describe('Exam Management Component', () => {
         // THEN
         expect(service.delete).to.have.been.calledOnce;
     });
+
+    it('Should return true for examHasFinished when component has no exam information ', () => {
+        // GIVEN
+        comp.examIdToExamInformation = new Map<number, ExamInformationDTO>();
+
+        // WHEN
+        const examHasFinished = comp.examHasFinished(exam.id!);
+
+        // THEN
+        expect(examHasFinished).to.be.true;
+    });
+
+    it('Should return true for examHasFinished when component has information of other exams', () => {
```

Comment:
I do not understand this test case? What do you mean by has information of other exams

Reply:
Ah what is meant is that I pass a non-empty `comp.examIdToExamInformation` to the function